packs/pfs-season-2-bestiary/chesjilawa-jadwiga-karina-5-6.json

```diff
@@ -228,8 +229,8 @@
                         "primal"
                     ],
                     "value": [
-                        "cold",
                         "concentrate",
+                        "fire",
```

Comment:
I don't believe the text states it should lose the fire trait and gain the cold trait.